package.json

```diff
@@ -3,9 +3,9 @@
   "type": "module",
   "version": "0.0.1",
   "scripts": {
-    "dev": "astro dev",
-    "start": "astro dev",
-    "build": "astro build",
+    "dev": "npm run script:extract-icon-names && astro dev",
```

Comment:
When this runs, it scans ubie-icon, extracts the filename, and outputs it to json